common-theurgy.lic

```diff
@@ -142,12 +142,24 @@ module DRCTH
     DRCI.put_away_item?(DRC.left_hand, container)
   end
 
+  def sprinkle_holy_water?(theurgy_supply_container, water_holder, target)
+      DRCI.get_item?(water_holder, theurgy_supply_container) &&
+      sprinkle?(water_holder, target) &&
+      DRCI.put_away_item?(water_holder, theurgy_supply_container)
+  end
+
   def sprinkle_holy_water(theurgy_supply_container, water_holder, target)
     DRCI.get_item?(water_holder, theurgy_supply_container)
     sprinkle?(water_holder, target)
     DRCI.put_away_item?(water_holder, theurgy_supply_container)
   end
 
+  def sprinkle_holy_oil?(theurgy_supply_container, target)
+      DRCI.get_item?('holy oil', theurgy_supply_container) &&
+      sprinkle?('oil', target) &&
+      DRCI.put_away_item?('holy oil', theurgy_supply_container) if DRCI.in_hands?('oil')
```

Comment:
Sorry, I didn't see this the first time I looked at the diff.

I don't think I've seen an `if` in a chained conditional like this... Have you tested this helper to make sure it puts away oil if it's in hand and skips (but returns true!) if not?

Reply:
How's the current version? I've tried to copy the format from `wave_incense?`